### PR TITLE
[service.subtitles.subscene] v1.7.2

### PR DESCRIPTION
**Pull request by thiagofigueiro**
### Description

When searching for subtitles for a TV show that contains more than 29 seasons, the error below can be seen in the logs:

```
12:08:36.047 T:1364964256   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.IndexError'>
                                            Error Contents: list index out of range
                                            Traceback (most recent call last):
                                              File "/storage/.kodi/addons/service.subtitles.subscene/service.py", line 532, in <module>
                                                search(item)
                                              File "/storage/.kodi/addons/service.subtitles.subscene/service.py", line 355, in search
                                                search_tvshow(item['tvshow'], item['season'], item['episode'], item['3let_language'], filename)
                                              File "/storage/.kodi/addons/service.subtitles.subscene/service.py", line 307, in search_tvshow
                                                search_string = tvshow + " - " + seasons[int(season)] + " Season"
                                            IndexError: list index out of range
                                            -->End of Python script error report<--
12:08:36.165 T:1858073504   ERROR: GetDirectory - Error getting plugin://service.subtitles.subscene/?action=search&languages=English&preferredlanguage=Unknown
```

Changes in this PR:
1. Fix the error above
2. Remove the use of reserved variable names, as per PEP8 and @learningit's [previous request](https://github.com/xbmc/repo-scripts/pull/201#issuecomment-228076865).
3. Bump version to 1.7.2

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

### CODE REVIEW

File: service.subtitles.subscene/service.py [hunk 1]
```diff
@@ -456,9 +460,9 @@ def download(link, episode=""):
 
         if len(subtitle_list) == 0:
             if episode:
-                xbmc.executebuiltin((u'Notification(%s,%s)' % (__scriptname__, __language__(32002))).encode('utf-8'))
+                xbmc.executebuiltin((u'Notification(%s,%s)' % (SCRIPT_NAME, LANGUAGE(32002))).encode('utf-8'))
```
Comment (enen92) [inline service.subtitles.subscene/service.py]: can you replace by `xbmcgui.Dialog().notification()`?

File: service.subtitles.subscene/service.py [hunk 2]
```diff
@@ -456,9 +460,9 @@ def download(link, episode=""):
...
             else:
-                xbmc.executebuiltin((u'Notification(%s,%s)' % (__scriptname__, __language__(32003))).encode('utf-8'))
+                xbmc.executebuiltin((u'Notification(%s,%s)' % (SCRIPT_NAME, LANGUAGE(32003))).encode('utf-8'))
```
Comment (enen92) [inline service.subtitles.subscene/service.py]: same

File: service.subtitles.subscene/service.py [hunk 3]
```diff
@@ -205,7 +208,7 @@ def getallsubs(url, allowed_languages, filename="", episode=""):
 
     codes = get_language_codes(allowed_languages)
     if len(codes) < 1:
-        xbmc.executebuiltin((u'Notification(%s,%s)' % (__scriptname__, __language__(32004))).encode('utf-8'))
+        xbmc.executebuiltin((u'Notification(%s,%s)' % (SCRIPT_NAME, LANGUAGE(32004))).encode('utf-8'))
```
Comment (enen92) [inline service.subtitles.subscene/service.py]: same

File: service.subtitles.subscene/service.py (hunk 1 above)
Comment (thiagofigueiro) [reply service.subtitles.subscene/service.py]: Thanks for the review. Done for all of them. 👍

**Comment by enen92** [comment]
Thanks for adjusting

**Pull request merged**